compiler/scenario-service/client/src/DA/Daml/LF/ScenarioServiceClient.hs

```diff
@@ -42,8 +48,8 @@ data Options = Options
   , optLogError :: String -> IO ()
   }
 
-toLowLevelOpts :: Options -> LowLevel.Options
-toLowLevelOpts Options{..} = LowLevel.Options{..}
+toLowLevelOpts :: Maybe Int -> Options -> LowLevel.Options
```

Comment:
Can we put the `Maybe Int` into the `Options`?

Reply:
That doesn’t really make sense since we read it from the config file here. We could move the logic for reading the config file somewhere else but I’m not sure propagating this from the top-level really helps makes things clearer.

Reply:
Alright. I don't have all the context on this anyway. I was just wondering but I'm happy to go your way.